vsintegration/VisualFSharpVsix/ShippingManifests/Desktop/Source.extension.vsixmanifest

```diff
@@ -5,10 +5,29 @@
     <Identity Id="VisualFSharp" Version="15.4.1.0" Language="en-US" Publisher="Microsoft.VisualFSharpTools" />
     <DisplayName>Visual F# Tools</DisplayName>
     <Description xml:space="preserve">Deploy Visual F# Tools templates to Visual Studio</Description>
+    <ShortcutPath>..\CommonExtensions\Microsoft\FSharp</ShortcutPath>
   </Metadata>
   <Installation>
     <InstallationTarget Id="Microsoft.VisualStudio.VSWinDesktopExpress" Version="[15.0]" />
   </Installation>
+  <Installer>
+    <Actions>
+      <Action Type="Ngen" Path="FsiAnyCPU.exe" />
```

Comment:
Does MicroBuild ensure that these are all signed before the vsix is made?

And is the resulting vsix signed?